tests/security/audit/autrace.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "autrace" utility traces individual processes in a fashion similar to strace. The output of autrace is logged to the audit log.
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub get_pid {
+    # Call this fuction to extract pid from autrace output in command line
+    script_run("tail -1 /tmp/out > /tmp/backup");
+    script_run("awk -F\\' '{print \$2}' /tmp/backup > /tmp/out");
+    my $pid = script_output('cat /tmp/out | cut -d " " -f 4');
+    return ($pid);
+}
+
+sub run {
+    my $audit_log = '/var/log/audit/audit.log';
+
+    select_console 'root-console';
+
+    # Install related packages and start audit service
+    zypper_call('in audit libaudit1');
+    assert_script_run("systemctl start auditd");
+
+    # Use autrace to trace an individual process, output will be logged to audit log
+    script_run('autrace /bin/ls /tmp');
```

Comment:
Please record_info the output of "autrace /bin/ls /tmp" for reference.
By default the outputs and return value are:
`# autrace /bin/ls /tmp
autrace cannot be run with rules loaded.
Please delete all rules using 'auditctl -D' if you really wanted to
run this command.
susetest:/tmp/apparmor.d # echo $?
1
susetest:/tmp/apparmor.d # 
`